.circleci/Dockerfile

```diff
@@ -4,6 +4,8 @@ RUN sudo apt update && sudo apt install -y libsdl2-dev libssl-dev && sudo apt cl
 
 RUN cargo install just --version ^0.6
 RUN cargo install cargo-readme --version ^3.2
+
+# MSRV: Upgrade to 0.6 when >= 1.45.0
```

Comment:
Yeah that would be good. I only added this comment because I saw the MSRV in the `deadlinks` readme. I did have a quick look at the `just` repo but couldn't find any versioning info there.